src/passes/RemoveNonJSOps.cpp

```diff
@@ -324,6 +285,57 @@ struct RemoveNonJSOpsPass : public WalkerPass<PostWalker<RemoveNonJSOpsPass>> {
   void visitGlobalGet(GlobalGet* curr) {
     neededImportedGlobals.insert(std::make_pair(curr->name, curr->type));
   }
+
+  void rewriteCopysign(Binary* curr) {
```

Comment:
Unless there was a good reason to move it, it would be good to put this function back where it was to keep the diff smaller. (It would be good to keep the new comment, though.)

Reply:
Ah, sure!